### PR TITLE
Add fuzzer for libchewing

### PR DESCRIPTION
**Pull request by kcwu**

### CODE REVIEW

**Comment by kcwu** [comment]
Ah, I found a problem. This fuzzer needs data files. 

With helper script's "run_ruzzer", it cannot find the data files and the coverage (cov) stop at 160.

**Comment by mikea** [comment]
Yes, you can install data files into /out. What's in it?

**Comment by kcwu** [comment]
Done. CL revised.

libchewing is a library for Chinese character input. It needs tables for "keystrokes to characters" conversion.

Now my fuzzer assumes the data files put at the same directory as the fuzzer executable.

File: libchewing/build.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+make -C test CFLAGS="$CFLAGS -Dmain=stress_main -Drand=get_fuzz_input" stress.o
+
+./libtool --mode=link \
+    $CC $CFLAGS \
+    -o /out/chewing_fuzzer \
+    /src/oss-fuzz/libchewing/chewing_fuzzer.c \
+    test/stress.o test/libtesthelper.la src/libchewing.la $LDFLAGS /work/libfuzzer/*.o
+
+# install data files
+make -C data pkgdatadir=/out install
```
Comment (mikea) [inline libchewing/build.sh]: This also creates /out/.libs file
Can these be removed?

Comment (kcwu) [inline libchewing/build.sh]: I will fix it tomorrow

**Pull request merged**